ProductPreviewer/Animation.swift

```diff
@@ -25,102 +24,45 @@ class Animation{
         print(maskDiameter) //893
         let mask = CAShapeLayer()
         let animationId = "path"
-
         // Make a circular shape.
         mask.path = UIBezierPath(roundedRect: CGRect(x: 0, y: 0, width: maskDiameter, height: maskDiameter), cornerRadius: maskDiameter / 2).cgPath
-
         mask.position = CGPoint(x: (view.bounds.width - maskDiameter) / 2, y: (view.bounds.height - maskDiameter) / 2)
-        
         // Add as a mask to the parent layer.
         view.layer.mask = mask
-        
-
         // Animate.
         let animation = CABasicAnimation(keyPath: animationId)
         animation.duration = duration
         animation.fillMode = .forwards
         animation.isRemovedOnCompletion = true
-
         // Create a new path.
         let newPath = UIBezierPath(roundedRect: CGRect(x: maskDiameter / 2, y: maskDiameter / 2, width: 0, height: 0), cornerRadius: 0).cgPath
-        
         if identifier == 1{
             // Set start and end values.
-            animation.fromValue = newPath //mask.path
-            animation.toValue = mask.path //newPath
+            animation.fromValue = newPath
+            animation.toValue = mask.path
         }
         else{
             animation.fromValue = mask.path
             animation.toValue = newPath
-            
         }
-
         // Start the animation.
         mask.add(animation, forKey: animationId)
         DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
         completion()
         }
     }
     
-    func changeBlue(imageView: UIImageView){
```

Comment:
Give it a better name changeBlue does not make any sense. Do it for all the other functions also.